src/utils/config.py

```diff
@@ -651,10 +651,19 @@ def _hydrate(self):
                         raw_msg=True,
                     )
             value = self.values[option["name"]]
+            
+            # Allow to use value instead of current_value in app config script.
```

Comment:
```suggestion
            # Allow to use value instead of current_value in app config script.
            # e.g. apps may write `echo 'value: "foobar"'` in the config file (which is more intuitive that `echo 'current_value: "foobar"'`
```